src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/PaginationDecorator.js

```diff
@@ -40,10 +41,13 @@ export default class PaginationDecorator extends React.PureComponent<Props> {
                         >
                             {children}
                         </InfiniteScroller>
-                        <div className={paginationDecoratorStyles.infiniteScrollLoader}>
+                        <div className={paginationDecoratorStyles.infiniteScrollIndicator}>
```

Comment:
I think the entire loading thing should go into the `InfiniteScroller`,  since this has to be used at any location the `InfiniteScroller` is used.